### PR TITLE
Make titles consistent

### PR DESCRIPTION
**Pull request by MrPotatoBobx**
Fixes https://github.com/elementary/installer/issues/307
 
 I have changed most options to start with a verb but i do not know how to word the "custom(advanced)" option

### CODE REVIEW

**Comment by MrPotatoBobx** [comment]
> Fixes #307
> 
> I have changed most options to start with a verb but i do not know how to word the "custom(advanced)" option

Perhaps  "do advanced install with custom partitioning"? but i think thats a bit detailed, id love to see your opinions,

One more thing, IMO, the text in tryinstallview("Try or install") should be changed to "Try or Install ElementaryOS"

**Comment by jeremypw** [comment]
I don't like "Do ..." for some reason.  "Perform ..." sounds better in this context.  But can you not start with the verb "Install"?

e.g. "Install cleanly"  "Install with custom partitioning".  May be better to explain what a "Clean Install" means in this context.

**Comment by jeremypw** [comment]
Yes, "cleanly" was just a placeholder for a more explanatory phrase.

**Comment by MrPotatoBobx** [comment]
Perhaps
"Install by Erasing Disk"
And 
"Install With Custom Partitioning"
?

**Comment by jeremypw** [comment]
Seems reasonable but needs input from the design team

File: src/Views/TryInstallView.vala [hunk 1]
```diff
@@ -78,7 +78,7 @@ public class Installer.TryInstallView : AbstractInstallerView {
         );
 
         var clean_install_button = new InstallTypeButton (
-            _("Clean Install"),
+            _("Do a Clean Install"),
```
Comment (danrabbit) [inline src/Views/TryInstallView.vala:81]: Maybe "Erase and Install" would be more clear

**Comment by MrPotatoBobx** [comment]
There we go, anything else?

File: src/Views/TryInstallView.vala [hunk 2]
```diff
@@ -78,13 +78,13 @@ public class Installer.TryInstallView : AbstractInstallerView {
...
         var clean_install_button = new InstallTypeButton (
-            _("Do a Clean Install"),
+            _("Erase Disk and Install"),
             "edit-clear",
             _("Erase everything and install a fresh copy of %s.").printf (Utils.get_pretty_name ())
         );
 
         var custom_button = new InstallTypeButton (
-            _("Custom (Advanced)"),
+            _("Insall with custom partitioning"),
```
Comment (danrabbit) [inline src/Views/TryInstallView.vala]: Small typo, and it looks like before we were using Title Case here
```suggestion
            _("Install with Custom Partitioning"),
```

**Review by danrabbit — CHANGES_REQUESTED** [review]

File: src/Views/TryInstallView.vala [hunk 3]
```diff
@@ -78,13 +78,13 @@ public class Installer.TryInstallView : AbstractInstallerView {
...
         var clean_install_button = new InstallTypeButton (
-            _("Clean Install"),
+            _("Erase Disk and Install"),
             "edit-clear",
             _("Erase everything and install a fresh copy of %s.").printf (Utils.get_pretty_name ())
         );
 
         var custom_button = new InstallTypeButton (
-            _("Custom (Advanced)"),
+            _("Install With Custom Partitioning"),
```
Comment (danrabbit) [inline src/Views/TryInstallView.vala:87]: This string is really long for the button label

**Comment by MrPotatoBobx** [comment]
The "custom install" thing is kind of hard to make it start wih a verb, would it kill us if we have one thing that isnt structured as the same of anything else?

File: src/Views/TryInstallView.vala (hunk 2 above)
Comment (MrPotatoBobx) [reply src/Views/TryInstallView.vala]: was fixed with https://github.com/elementary/installer/pull/510/commits/ac65c2533590b18c030779fbf20313423ec821cc

**Review by danrabbit — APPROVED** [review]

**Pull request merged**